examples/storm-elasticsearch-examples/pom.xml

```diff
@@ -62,6 +62,23 @@
                 <artifactId>maven-shade-plugin</artifactId>
                 <configuration>
                     <createDependencyReducedPom>true</createDependencyReducedPom>
+                    <filters>
```

Comment:
I'm wondering if it would work to put this in PluginManagement in the root pom instead?